dj_cqrs/transport/rabbit_mq.py

```diff
@@ -168,11 +172,34 @@ def _declare_exchange(channel, exchange):
         )
 
     @staticmethod
-    def _get_common_settings():
-        host = settings.CQRS.get('host', ConnectionParameters.DEFAULT_HOST)
-        port = settings.CQRS.get('port', ConnectionParameters.DEFAULT_PORT)
-        user = settings.CQRS.get('user', ConnectionParameters.DEFAULT_USERNAME)
-        password = settings.CQRS.get('password', ConnectionParameters.DEFAULT_PASSWORD)
+    def _parse_url(url):
+        scheme = urlparse(url).scheme
+        schemeless = url[len(scheme) + 3:]
+        parts = urlparse('http://' + schemeless)
+        path = parts.path or ''
+        path = path[1:] if path and path[0] == '/' else path
+        assert scheme == 'amqp', \
+            'Scheme must be "amqp" for RabbitMQTransport.'
+        return (
+            unquote(parts.hostname or '') or ConnectionParameters.DEFAULT_HOST,
+            parts.port or ConnectionParameters.DEFAULT_PORT,
+            unquote(parts.username or '') or ConnectionParameters.DEFAULT_USERNAME,
+            unquote(parts.password or '') or ConnectionParameters.DEFAULT_PASSWORD,
+        )
+
+    @classmethod
+    def _get_common_settings(cls):
+        if 'url' in settings.CQRS:
+            host, port, user, password = cls._parse_url(settings.CQRS.get('url'))
+        else:
+            logger.warning(
```

Comment:
I have some doubts on this deprecation warning. I wouldn't want to get it in places, where we use it.
Maybe, it makes sense to just remove logging, as the way is still okay.

Other then that, everything is PERFECT!

Reply:
:thumbsup: